### PR TITLE
update_chat_notifications new high-level method

### PR DESCRIPTION
**Pull request by CyanBook**
update_chat_notifications is now a high-level method

- it has bound methods for chat object
- it supports unix time, datetime and timedelta

### CODE REVIEW

File: pyrogram/methods/chats/update_chat_notifications.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from pyrogram.scaffold import Scaffold
+
+
+class UpdateChatNotifications(Scaffold):
+    async def update_chat_notifications(
+        self,
+        chat_id: Union[int, str],
+        show_previews: bool = None,
+        silent: bool = None,
+        mute_until: Union[int, datetime, timedelta] = None
```
Comment (delivrance) [inline pyrogram/methods/chats/update_chat_notifications.py:15]: This adds inconsistencies with the rest of the API, which currently only allows unix timestamps to be passed as argument.

**Review by delivrance — CHANGES_REQUESTED** [review]

File: pyrogram/methods/chats/update_chat_notifications.py (hunk 1 above)
Comment (CyanBook) [reply pyrogram/methods/chats/update_chat_notifications.py]: Now it accepts only unix time

**Review by delivrance — APPROVED** [review]
Ok as an initial support. I'm going to round some edges and maybe add a new convenience type as we probably need it.

**Comment by mrksbgg** [comment]
god bless pyro devs 🙏

**Comment by arynyklas** [comment]
no way

**Comment by rojvv** [comment]
Don't trash the comments so bad that Dan won't touch Pyrogram for another two years.

**Comment by ALiwoto** [comment]
wow amazing

**Comment by KurimuzonAkuma** [comment]
Time to finish with my fork?

**Comment by asnct** [comment]
yay

**Pull request closed without merging**